integrasjonstest/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/integrasjonstest/utils/EndToEndTest.kt

```diff
@@ -106,26 +107,40 @@ val maxMekker =
 abstract class EndToEndTest : ContainerTest() {
     private val imTestRapid = ImTestRapid()
 
+    // Vent på inntektsmeldingdatabase
     private val inntektsmeldingDatabase by lazy {
         println("Database jdbcUrl for im-db: ${postgresContainerOne.jdbcUrl}")
-        postgresContainerOne
-            .toHikariConfig()
-            .let(::Database)
-            .also {
-                val migrationLocation = Path("../db/src/main/resources/db/migration").absolutePathString()
-                it.migrate(migrationLocation)
-            }.createTruncateFunction()
+        repeat(5) {
+            runCatching {
+                postgresContainerOne
+                    .toHikariConfig()
+                    .let(::Database)
+                    .also {
+                        val migrationLocation = Path("../db/src/main/resources/db/migration").absolutePathString()
+                        it.migrate(migrationLocation)
+                    }.createTruncateFunction()
+            }.onSuccess { return@lazy it }
+                .onFailure { runBlocking { delay(1000) }.also { println("Prøver igjen å sette opp testdatabase.") } }
+        }
+        throw IllegalStateException("Klarte ikke sette opp inntektsmeldingDatabase.")
```

Comment:
🏎️ 

Vi kunne kanskje laget en egen funksjon for den repeat-koden, siden den brukes flere steder nå.